src/java/src/main/java/triton/client/InferenceServerClient.java

```diff
@@ -347,7 +347,7 @@ public static class InferArguments {
         int sequenceId = 0;
         boolean sequenceStart = false;
         boolean sequenceEnd = false;
-        int priority = 0;
+        long priority = 0;
```

Comment:
I would like to see this ideally as `uint64_t` but if only using primitives then I would like to see this be `long long` to avoid any [ambiguity when it comes to size](https://en.cppreference.com/w/cpp/language/types).

Reply:
For java specifically, the equivalent is `long`